reactive/aws.py

```diff
@@ -52,26 +45,44 @@ def handle_requests():
     aws = endpoint_from_flag('endpoint.aws.requested')
     for request in aws.requests:
         if request.instance_tags:
-            tag_instance(request.instance_id,
-                         request.region,
-                         request.instance_tags)
-        if request.instance_security_group_tags:
-            tag_instance_security_group(request.instance_id,
-                                        request.region,
-                                        request.instance_security_group_tags)
+            charm_lib.tag_instance(
+                request.instance_id,
+                request.region,
+                request.instance_tags)
+        if request.unit_security_group_tags:
+            charm_lib.tag_unit_security_group(
+                request.instance_id,
+                request.region,
+                request.unit_security_group_tags)
         if request.instance_subnet_tags:
-            tag_instance_subnet(request.instance_id,
-                                request.region,
-                                request.instance_subnet_tags)
+            charm_lib.tag_instance_subnet(
+                request.instance_id,
+                request.region,
+                request.instance_subnet_tags)
         if request.requested_elb:
-            enable_elb(request.instance_id, request.region)
+            charm_lib.enable_elb(
+                request.application_name,
+                request.instance_id,
+                request.region)
         if request.requested_ebs:
-            enable_ebs(request.instance_id, request.region)
+            charm_lib.enable_ebs(
+                request.application_name,
+                request.instance_id,
+                request.region)
         if request.requested_route53:
-            enable_route53(request.instance_id, request.region)
+            charm_lib.enable_route53(
```

Comment:
i think you missed app_name in this method def (looks like it only takes 2 args atm).